@navikt/core/tokens/darkside/figma/plugin/AkselVariablesInterface.ts

```diff
@@ -0,0 +1,258 @@
+import _config from "../../../figma-config.json";
+import { FigmaConfigEntry, FigmaTokenConfig } from "../figma-config.types";
+import { FigmaPluginInterface } from "./FigmaPluginInterface";
+
+type ScopedFigmaTokenConfig = Omit<FigmaTokenConfig, "version" | "timestamp">;
+
+export class AkselVariablesInterface {
+  private Figma: FigmaPluginInterface;
+  private config: FigmaTokenConfig;
+  private meta: Pick<FigmaTokenConfig, "version" | "timestamp">;
+  private remoteConfigURL =
+    "https://cdn.nav.no/designsystem/@navikt/tokens/figma-config.json";
+
+  constructor() {
+    const config = _config as FigmaTokenConfig;
+    this.config = config;
+    this.meta = { timestamp: config.timestamp, version: config.version };
+    this.Figma = new FigmaPluginInterface();
+  }
+
+  /**
+   * Because of "dynamic-page" loading in Figma,
+   * we need to initialize the plugin trough async methods.
+   * This method should be called before any other methods.
+   */
+  async init(): Promise<void> {
+    await this.Figma.init();
+  }
+
+  async useRemoteConfig(): Promise<void> {
+    const newConfig = await fetch(this.remoteConfigURL)
+      .then((res) => res.json())
+      .catch((err) => {
+        console.error(err);
+        throw new Error("Error fetching config from CDN 😱");
+      });
+    this.meta = { timestamp: newConfig.timestamp, version: newConfig.version };
+
+    this.config = newConfig;
+  }
+
+  updateVariables(): void {
+    this.updateGlobalColorCollection(this.config.colors.light.global);
+    this.updateGlobalColorCollection(this.config.colors.dark.global);
+    this.updateSemanticColorCollection(this.config.colors);
+    this.updateScaleCollection(this.config.radius);
+    this.updateScaleCollection(this.config.spacing);
+    console.info("Updated variables!");
```

Comment:
```suggestion
    console.info("Variables updated!");
```¨
Donno y, but this sounds more correct